curvefs/src/metaserver/dentry_storage.cpp

```diff
@@ -133,13 +133,19 @@ void DentryList::PushBack(DentryVec* vec) {
         return;
     } else if (dentrys.size() == 0 || HasDeleteMarkFlag(*last)) {
         return;
-    } else if (onlyDir_ && last->type() != FsFileType::TYPE_DIRECTORY) {
-        return;
     } else if (last->name() == exclude_) {
         return;
     }
 
     size_++;
+
+    if (onlyDir_ && last->type() != FsFileType::TYPE_DIRECTORY) {
```

Comment:
meybe we should log the seek time after list done to provide the important information for troubleshooting.

Reply:
fixed